lib/Fmt_ast.ml

```diff
@@ -178,6 +178,10 @@ let update_config_maybe_disabled c loc l f =
   let c = update_config c l in
   maybe_disabled c loc l f
 
+let update_config_maybe_disabled_attrs c loc attrs f =
+  let l = attrs.attrs_before @ attrs.attrs_after in
+  maybe_disabled c loc l f
```

Comment:
This causes the bug in `test/passing/tests/protected_object_types.ml`
```suggestion
  update_config_maybe_disabled c loc l f
```

Reply:
DOne !